apps/sports-helsinki/.env.local.example

```diff
@@ -1,32 +0,0 @@
-NEXT_PUBLIC_FEDERATION_ROUTER_ENDPOINT="https://events-graphql-federation-sports.test.hel.ninja/"
```

Comment:
This is the example file for local private environment config. This should not be deleted!

- `.env.local.example` is there for developers. Its needed. Note the dot-prefix.
- `.env.local` is in use, but it's in gitignore, because it's a private local file. Note the dot-prefix.


There should not be any other env-related files at the moment.